### PR TITLE
Create README.md

### LINKED ISSUE #4: Create README.md
Create README.md file specifying repository purpose, lightweight structural description, link to wiki, contributions, licensing and authorship.

Requirement Reference: 1

### PR DESCRIPTION
**Pull request by samuelsogbesan**
This PR integrates the initial README.md file.
The structure of the file is as follows:
- **Project Description** - Light description of project purpose
- **Wiki** - Explanation of what will be seen on wiki, providing link to it
- **Contributions** - Detail who has contributed to the project
- **Licensing** - Information on licensing agreements, and justifications.

Closes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+A lightweight dashboard designed to display the build status of LLVM Build Bots in an easily interpretable fashion via a User Interface.
+
+## Repository Structure
+
+
+## Wiki
+[The wiki](https://github.com/googleinterns/step240-2020/wiki) describes the purpose of different parts of the system.
+
+## Contributions
+The whole of the Graph Geckos Team.
```
Comment (MagoDelBlocco) [inline README.md:13]: nit: list out the members of the team

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# LLVM Build Dashboard
+
+## Project Description
+A lightweight dashboard designed to display the build status of LLVM Build Bots in an easily interpretable fashion via a User Interface.
```
Comment (MagoDelBlocco) [inline README.md:4]: nit: This is personal preference, but I like [line wrapping documentation](https://about.gitlab.com/blog/2016/10/11/wrapping-text/) (consider 80 character line wraps).

**Review by MagoDelBlocco — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (samuelsogbesan) [reply README.md]: Lines wrapper here 284f850

File: README.md (hunk 1 above)
Comment (samuelsogbesan) [reply README.md]: Names listed 7f97e4b

File: README.md (hunk 2 above)
Comment (MagoDelBlocco) [reply README.md]: ditto

`A lightweight dashboard designed to display the build status of LLVM Build Bots in an easily interpretable` has 107 characters, which does not adhere to any line wrap convention

Comment (samuelsogbesan) [reply README.md]: Updated here a3cd071

**Review by MagoDelBlocco — APPROVED** [review]
LGTM

**Review by MagoDelBlocco — APPROVED** [review]
LGTM

File: README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+A lightweight dashboard designed to display the build status of LLVM Build Bots 
+in an easily interpretable fashion via a User Interface.
+
+## Wiki
+[The wiki](https://github.com/googleinterns/step240-2020/wiki) describes the purpose of different parts of the
+system.
+
+## Contributions
+**The Graph Geckos Team**.
+Meike Baumgärtner
```
Comment (Somae) [inline README.md:13]: I suggest alphabetical order here for names.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+Krasimir Georgiev
+Samuel Sogbesan
+Bogdan Ciobanu
+Masha Samsikova
+
+## Licensing
+Licensed under Apache 2.0, inline with the licensing of the [LLVM Project](https://github.com/llvm/llvm-project)
```
Comment (Somae) [inline README.md:20]:
```suggestion
Licensed under Apache 2.0, in line with the licensing of the [LLVM Project](https://github.com/llvm/llvm-project).
```

**Review by Somae — APPROVED** [review]
Added some nits. I haven't checked specifically, but please make sure to stay within the character per line limits.

**Pull request merged**